carbon-design-system/carbon
# fix(a11y): Fixing tooltip accessibility on Window High Contrast Mode

Closes #6896

Fixing Tooltip accessibility  on Window High Contrast Mode

#### Changelog

**Changed**

Tooltip component

#### Testing / Reviewing

On a Windows 10 machine, go to `settings --> ease of access --> high contrast` and flick the switch. Compare the storybook in Edge (Should be in High Contrast) to Chrome (Should look normal). 
For **Firefox**, you'll need to open it up and type `about:config` into the address bar. Then, search for `layout.css.prefers-contrast` and set it to `true`.

- Visit [Tooltip](http://192.168.1.158:9000/?path=/story/tooltip--default-bottom) component
- Verify that the icon shows properly
- Verify the focus state works on the icon
- Verify that there's a border on the tooltip dialog
